### PR TITLE
adapt tangoAttrZeroDController for Sardana v3.03

### PR DESCRIPTION
**Pull request by dschick**

### CODE REVIEW

**Comment by reszelaz** [comment]
@rhomspuron could you please review this one? Thanks!

File: sardana_tango/ctrl/TangoAttrZeroDCtrl.py [hunk 1]
```diff
@@ -68,32 +68,48 @@ def pre_read_one(self, axis):
...
+                        dev_attr_value = values[attr]
+                        if dev_attr_value.has_failed:
+                            # In case of Attribute error
+                            VALUE = tango.DevFailed(*dev_attr_value.get_err_stack())
+                            self.devsExtraAttributes[axis][EVALUATED_VALUE] = VALUE
+                        else:
+                            formula = self.devsExtraAttributes[axis][FORMULA]
+                            VALUE = float(dev_attr_value.value)
+                            value = VALUE # just in case 'VALUE' has been written in lowercase...
+                            v = eval(formula)
```
Comment (rhomspuron) [inline sardana_tango/ctrl/TangoAttrZeroDCtrl.py:109]: You can change to:
value = float(dev_attr_value.value)
v = eval(formula, {'value': value, 'VALUE': value, 'Value':value}

**Review by rhomspuron — CHANGES_REQUESTED** [review]
Hi @dschick, 

Thanks for the PR. 

The ReadTangoAttribute class is used on the TangoAttrCTCtrl. I propose you to create a module: tango_reader.py where you move the class ReadTangoAttributes and use the same for both controller, in this way we have a common.

**Comment by rhomspuron** [comment]
Hi @dschick,

I will integrate the PR and add a TODO for the changes. I do not have permission to your branch, I will create another PR with your changes.

**Pull request closed without merging**